Libplanet.Explorer.Tools/Program.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Cocona;
+using Libplanet.Explorer.Tools.SubCommand;
+
+namespace Libplanet.Explorer.Tools
+{
+    [HasSubCommands(typeof(MySqlStore), Description = "Manage MySql store.")]

```

Comment:
If you wanted to say `MySqlStore`, how about the full name, `MySqlStore`? Or using `MySQL store`?